apps/careeropportunity/filters.py

```diff
@@ -0,0 +1,32 @@
+from django_filters import filters, filterset
+from watson import search as watson_search
+
+from .models import CareerOpportunity
+
+
+class CharInFilter(filters.BaseInFilter, filters.CharFilter):
+    pass
```

Comment:
Is this necessary?

Reply:
Yes, it is kinda strange but it essentially means: _combine these classes together, without adding anything else_

Reply:
Aha! I see it now, thanks!